js/csv.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Converts a JSON object to a valid CSV string file.
+ *
+ * @param {Object} object The object to convert.
+ * @returns {String} The converted string.
+ */
+export const objectToCsv = object => {
+    let csv = "";
+    if (Object.keys(object) === 0) return csv;
+
+    const fields = Object.keys(object[0]);
+    csv += fields.toString() + "\n";

```

Comment:
Is this the header? If so, I think we usually separate it with commas. Maybe add an optional parameter to the function that defaults to `,` to be used as a delimiter.